src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
```

Comment:
Don't create a new instance of Random every time the method is called. Instead, make it a private class-level field to reuse the same instance.